yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtText.kt

```diff
@@ -75,14 +75,32 @@ internal data class CrdtText(
         range: RgaTreeSplitPosRange,
         attributes: Map<String, String>,
         executedAt: TimeTicket,
-    ): List<TextChange> {
+        latestCreatedAtMapByActor: Map<ActorID, TimeTicket>? = null,
+    ): Pair<Map<ActorID, TimeTicket>, List<TextChange>> {
```

Comment:
I think the return type is getting too complex now 🤔
It would be better to implement a custom class.